app/scripts/controllers/tesoreria/avances/solicitud/lista_solicitud.js

```diff
@@ -164,46 +164,71 @@ angular.module('financieraClienteApp')
                 }
             ]
         };
-
         ctrl.ver_fila = function(row) {
```

Comment:
es necesario generalizar nombre de la variable asignada al this,  entre self o ctrl. 
dado que self es utilizado en la mayoria de modulos sera este el que se utilice.
se debe cambiar nombre de la variable.